packages/components/pie-modal/test/component/pie-modal.spec.ts

```diff
@@ -83,10 +93,162 @@ test.describe('`Pie Modal is closed`', () => {
                 },
             );
 
+            // Act
             await page.locator('#dialog').click();
 
+            // Assert
             expect(messages).toHaveLength(1);
         });
     });
 });
 
+test.describe('Pie Modal `isDismissible`', () => {
+    test.describe('when `isDismissible` is truthy', () => {
```

Comment:
I think it should be 

"when `isDismissible` is true"

Truthy implies 'a', 2 or true would all be valid values.

Reply:
Fixed.